### DISCUSSION TITLE
Any luck on ghs.googlehosted.com

### DISCUSSION THREAD

**Discussion by 0xcrypto** [EdOverflow/can-i-take-over-xyz#201] [Q&A] [answered]
Is it possible to takeover sites pointed to ghs.googlehosted.com?

**Comment by indianajson** [comment, marked as answer]
Based on my experience, I believe takeovers of (sub)domains with CNAMEs pointing to `ghs.googlehosted.com` are **no longer possible** since Google now requires domain ownership verification.

**Reply by zulficar770-jpg** [reply]
Hi

**Comment by med0101z** [comment]
any news about ghs.googlehosted.com takeover ??
